.eslintrc

```diff
@@ -77,6 +77,14 @@ rules:
   ## Suggest using 'const' wherever possible
   prefer-const: 2
 
+  # Arrow Functions
+  ## Enforce parens around arrow function arguments
+  arrow-parens: [2, "always"]
+  ## Require a space on each side of arrow operator
+  arrow-spacing: [2, { "before": true, "after": true }]
```

Comment:
Interesting, I wasn't aware that [quotes are optional in YAML](http://stackoverflow.com/a/22235064/808699). I'll investigate if we can remove all quotes.